vtr_flow/scripts/run_vtr_flow.pl

```diff
@@ -118,6 +120,7 @@
 my $check_route             = 0;
 my $check_place             = 0;
 my $use_old_abc_script      = 0;
+my $check_incremental       = 0;
```

Comment:
`check_incremental` is a bit generic. I'd suggest something a bit more explicit, like `check_incremental_timing_consistency`. Similarly for the actual command-line option below.